code/modules/mob/living/silicon/decoy/decoy.dm

```diff
@@ -6,12 +6,6 @@
 	canmove = 0
 	a_intent = INTENT_HARM // This is apparently the only thing that stops other mobs walking through them as if they were thin air.
 
-/mob/living/silicon/decoy/New()
-	src.icon = 'icons/mob/ai.dmi'
-	src.icon_state = "ai"
-	src.anchored = 1
-	src.canmove = 0
```

Comment:
It appears that when you make this call parent it makes an issue with huds

Reply:
`canmove = 0`
Is that a boolean?!